ref/general/what-is-open-liberty.adoc

```diff
@@ -16,26 +16,35 @@
 
 Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
 
-With Open Liberty, you can add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE 8 Full Platform and Web Profile specifications and with MicroProfile 3.0. Open Liberty is deployed on every major developer platform, including Docker, Kubernetes, and Cloud Foundry. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
+With Open Liberty, you can add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE Full Platform and Web Profile specifications and with MicroProfile. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
 
 Open Liberty offers developers the following benefits:
+
 * A lightweight edition that corresponds to the Java EE Web Profile specification.
 * Excellent development and production runtime environments for web applications.
 * A smaller footprint for faster download and startup, giving more development time and faster time to deployment.
 * Ease of packaging applications for deployment, including configuration.
 * The ability to run applications that are written for Open Liberty on WebSphere Application Server traditional.
 * The ability to extend the Open Liberty capabilities by adding custom features that use a product extension System Programming Interface (SPI).
 
+== Zero migration
+
+With the Open Liberty zero migration policy, you focus on writing your applications, not the APIs changing under you. Zero-migration architecture means that you can use existing, unmodified configuration and application files with an updated version of Open Liberty without unwanted or unexpected change in behavior. Open Liberty offers full compatibility between product versions and pluggable features so that your existing APIs and behaviors are supported in each new product version.
```

Comment:
Change,
"Open Liberty offers full compatibility between product versions and pluggable features so that your existing APIs and behaviors are supported in each new product version."
to
"Open Liberty offers full compatibility between product versions and pluggable features**,** so that your existing APIs and behaviors are supported in each new product version."

Reply:
I had a comma here but acrolinx said to remove it